pyrenew/latent/hospitaladmissions.py

```diff
@@ -210,11 +214,11 @@ def sample(
             *_,
         ) = self.infection_to_admission_interval_rv(**kwargs)
 
-        latent_hospital_admissions = jnp.convolve(
-            infection_hosp_rate.value * latent_infections.value,
+        latent_hospital_admissions = compute_delay_ascertained_incidence(
```

Comment:
Looks like this has unapproved/unmerged changes from https://github.com/CDCgov/PyRenew/pull/356. These should be removed or we should wait until https://github.com/CDCgov/PyRenew/pull/356 is merged.

Reply:
#356 is ready for review, let's wait until it's merged